compiler/one-cmds/tests/partition_esrgan-add.test-x

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
I think.... it is 2021
`2020 Samsung Electronics Co ` >> `2021 Samsung Electronics Co`